spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -127,11 +127,26 @@
       end
     end
 
-    context 'when prompt is not select_account' do
+    context 'when prompt is not select_account or login' do
       let(:prompt) { 'aaa' }
       it { expect(valid?).to eq(false) }
     end
 
+    context 'when prompt is not given' do
+      let(:prompt) { nil }
+      it { expect(valid?).to eq(true) }
+    end
```

Comment:
Is prompt not a required parameter in OIDC?

Reply:
That is correct.  Prompt will no longer be required per the new requirements: https://github.com/18F/identity-private/issues/2516